From 4cca6cf5aa5a2a7567bc18d27127557f2a737c0f Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Wed, 12 Aug 2015 14:53:58 +0300
Subject: [PATCH 12/12] imx6sl-evk: update wilink8 entries for r8.6

Due to changes in upstream there are a couple of device tree related
parameters that are needed for wilink8 initialization that have changed

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 arch/arm/boot/dts/imx6sl-evk-common.dtsi |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sl-evk-common.dtsi b/arch/arm/boot/dts/imx6sl-evk-common.dtsi
index 1e7f7ce..a797743 100644
--- a/arch/arm/boot/dts/imx6sl-evk-common.dtsi
+++ b/arch/arm/boot/dts/imx6sl-evk-common.dtsi
@@ -5,6 +5,7 @@
  * it under the terms of the GNU General Public License version 2 as
  * published by the Free Software Foundation.
  */
+#include <dt-bindings/interrupt-controller/irq.h>
 
 / {
 	memory {
@@ -521,10 +522,9 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 	wlcore: wlcore@0 {
-		compatible = "ti,wlcore";
+		compatible = "ti,wl1835";
 		reg = <2>;
 		interrupt-parent = <&gpio5>;
-		interrupts = <8 0>;
-		platform-quirks = <1>;
+		interrupts = <8 IRQ_TYPE_EDGE_RISING>;
 	};
 };
-- 
1.7.9.5

